kevlar/include/kevlar_bits/driver/accumulate.hpp

```diff
@@ -40,17 +40,15 @@ inline void accumulate(const SGSType& sgs, const GridRangeType& grid_range,
     }
 
     if (n_threads > max_threads) {
-        n_threads %= max_threads;
+        n_threads = max_threads;
```

Comment:
Modulo behavior seemed surprising to me. If n_threads = 9 and max_threads = 8, why should we set to n_threads = 1?

Reply:
Oh cuz I'm dumb af lmao. This is definitely what I intended